README.md

```diff
@@ -1,19 +1,34 @@
 # ImageDePHI
+ImageDePHI is an application that can be used to redact PHI from whole slide images.
+
 
 This project has been funded in whole or in part with Federal funds from the National Cancer Institute, National Institutes of Health, Department of Health and Human Services, under Contract No. 75N91022C00033
 
-## To Run as a CLI
 
-`pip install -r requirements.txt`
+# Installation
+ImageDePHI is under active development and as such the best way to test the latest features is to download the most current artifact.
```

Comment:
https://github.com/DigitalSlideArchive/ImageDePHI/releases/latest will take you to the latest release